src/Compilers/VisualBasic/Test/Emit/Attributes/AttributeTests_WellKnownAttributes.vb

```diff
@@ -5127,6 +5127,153 @@ End Class
         End Sub
 #End Region
 
+#Region "SkipLocalsInitAttribute"
+
+        <Fact>
+        Public Sub SkipLocalsInitAttributeOnMethod()
+            Dim source = <compilation>
+                             <file name="a.vb">
+                                 <![CDATA[
+Namespace System.Runtime.CompilerServices
+    Class SkipLocalsInitAttribute
+        Inherits System.Attribute
+    End Class
+End Namespace
+
+Class C
+    <System.Runtime.CompilerServices.SkipLocalsInitAttribute>
+    Sub M()
+    End Sub
+End Class
+]]>
+
+                             </file>
+                         </compilation>
+
+            Dim comp = CreateCompilationWithMscorlib40(source)
+
+            comp.VerifyDiagnostics(
```

Comment:
>comp.VerifyDiagnostics( [](start = 12, length = 23)

Please use AssertTheseDiagnostics helper instead (see examples in this file). This comment applies to all added tests

Reply:
I can do that, but is there any particular reason to use it instead of VerifyDiagnostics?